labs/basics/src/main.cpp

```diff
@@ -0,0 +1,14 @@
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+#include "reviser.hpp"
+
+int main(int argc, char* argv[]) {
+    QGuiApplication app(argc, argv);
+    QQmlApplicationEngine engine;
+
+    qmlRegisterType<Reviser>("Reviser.Reviser", 1, 0, "Reviser");
+
+    const QUrl url("qrc:/path/qml/main.qml");
```

Comment:
Ну вообще так, и всё будет работать тоже самое можно сделать и в ambience, и тогда под Bazel'ем можно будет всё красиво собрать (или посмотреть фикс правил здесь https://github.com/pkasila/rules_qt6/pull/1 - уже в ветке, так что можно использовать... только мне обновить коммит в MODULE.bazel, или можешь сам :D)

```suggestion
    const QUrl url("qrc:labs/basics/qml/main.qml");
```

Reply:
Проблема была в том, что пример сборки qml в bazel подключает _весь_ Qt как зависимость, и это тянет ещё много зависимостей, которые не включены в бинарную сборку Qt и линкуются динамически. У меня например нет этой либы:

```
error while loading shared libraries: libpcsclite.so.1: cannot open shared object file: No such file or directory
```

и ещё кучу других либ, в том числе из состава Qt. В чём тогда смысл этой бинарной сборки, если она не заводится по команде `bazel run` и требует самому доустановить кучу библиотек (в том числе и ненужных для работы приложения) в систему?